### DISCUSSION TITLE
mixcr-mixcr analyze 10x-sc-xcr-vdj

### DISCUSSION THREAD

**Discussion by whitebird01** [milaboratory/mixcr#1839] [General]
Thank you for the development of the Mixcr software. When I run the 10X single-cell VDJ data using the command:
`mixcr analyze 10x-sc-xcr-vdj --species mmu ...`
I noticed that the generated default YAML file contains the tag pattern:
`--tag-pattern "^(CELL:N{16})(UMI:N{10})\\^(R2:*)"`
However, I believe it should be (for 10X V3):
`--tag-pattern "^(CELL:N{16})(UMI:N{12})\\^(R2:*)"`
```
 mixcr analyze 10x-sc-xcr-vdj \
     --tag-pattern "^(CELL:N{16})(UMI:N{12})\\^(R2:*)" \
      --species mmu \
```

I would like to ask if there is any issue with this setting. Thank you very much!

10X V2："^(CELL:N{16})(UMI:N{10})\\^(R2:*)" \
10X V3："^(CELL:N{16})(UMI:N{12})\\^(R2:*)" \
![QQ图片20241029143950](https://github.com/user-attachments/assets/c6259bcd-3130-4c52-81db-58fcab01fcfd)

**Comment by whitebird01** [comment]
mixcr version: 4.7.0,repseqio.v5.1

**Comment by mizraelson** [comment]
Hi, 10x VDJ library is usually based on v2 chemistry, unless you have some custom solution.

**Comment by whitebird01** [comment]
Thank you very much for your reply. I understand what you mean, and I have further confirmed the version of the VDJ kit used in the wet experiment. Thanks again!
